ui/snippets/walletMenu/WalletMenuDesktop.tsx

```diff
@@ -21,12 +22,25 @@ type Props = {
   size?: 'sm' | 'md';
 };
 
-const WalletMenuDesktop = ({ isHomePage, className, size = 'md' }: Props) => {
-  const { isWalletConnected, address, connect, disconnect, isModalOpening, isModalOpen, openModal } = useWallet({ source: 'Header' });
+type ComponentProps = Props & {
+  isWalletConnected: boolean;
+  address: string;
+  connect: () => void;
+  disconnect: () => void;
+  isModalOpening: boolean;
+  isModalOpen: boolean;
+  openModal: () => void;
+};
+
+export const WalletMenuDesktopComponent = ({
```

Comment:
I don't think that `WalletMenuDesktopComponent` is a good name.
Maybe it is better to keep `WalletMenuDesktop` name and use `WalletMenuDesktopWrapper` for the parent component.